packages/stylelint-plugin/src/stylelintPlugin.spec.ts

```diff
@@ -660,31 +830,30 @@ describe("Codemod", () => {
     testName,
     input,
     expectedOutput,
-    expectedUnfixables,
+    expectedWarnings,
     only,
   }: TestExample) => {
     const testFn = only ? test.only : test
-    testFn(`${language}: ${testName}`, () => {
-      let unfixables = 0
-      const result = codemodOnSource(input, {
-        language,
+    testFn(`${language}: ${testName}`, async () => {
+      const result = await lint({
+        codeFilename: `input.${language}`,
+        config: testStylelintConfig,
+        code: input,
         fix: true,
-        reporter: ({ message, autofixAvailable }) => {
-          // Uncomment this if you want to see reports
-          // console.log(message)
-          if (!autofixAvailable) {
-            unfixables++
-          }
-        },
       })
+      if (result.results[0]?.warnings.length !== expectedWarnings) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          `Unexpected warnings for test: ${language}: ${testName}`,
+          result.results[0]?.warnings
+        )
+      }
```

Comment:
This will tell you what was reported if there is a mismatch in expected warnings.
The `expect` doesn't tell you anything about the reason behind the warnings.
Still think it should go? I think it helps to debug failing tests.

Reply:
Good point - I saw you do this elsewhere and think it can make sense, but forgot to come back to this comment